tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/ClientApplications/NativeDesktop/UpdateNativeDesktopMemoryAndStorageTests.cs

```diff
@@ -199,9 +199,7 @@ public void InvalidSolutionIdShouldThrowNotFound()
 
             Expression<Func<ISolutionRepository, Task>> solutionRepositoryExpression = r =>
                 r.ByIdAsync(SolutionId, It.IsAny<CancellationToken>());
-
```

Comment:
I think we should keep this blank line – I think it makes the code slightly more readable.

Reply:
Added blank line.